src/mlpack/methods/ann/layer/celu.hpp

```diff
@@ -0,0 +1,203 @@
+/**
+ * @file celu.hpp
+ * @author Gaurav Singh
+ *
+ * Definition of the CELU activation function as descibed by Jonathan T. Barron.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_CELU_HPP
+#define MLPACK_METHODS_ANN_LAYER_CELU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The CELU activation function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) &=& \left\{
+ *   \begin{array}{lr}
+ *    x & : x >= 0 \\
+ *    \alpha(e^(x/alpha) - 1) & : x \le 0
```

Comment:
I'm thinking \le stands for less than or equal to. So I think you need to change it to just "less than" (since x>=0 already accounts for x=0). So I would suggest changing it to `x < 0`

Additionally, although optional, I would suggest another style change for the same line to the following:
`\alpha(e^(\frac{x}{\alpha}) - 1) & : x < 0`